main.go

```diff
@@ -28,6 +28,7 @@ var (
 	terminate = flag.Bool("terminate", false, "Terminate public services immediately.")
 	slackToken = flag.String("slack-token", "", "Slack API token to send notifications.")
 	slackChan = flag.String("slack-channel", "", "Slack channel to notify when terminating services.")
+	clusterId = flag.String("cluster-id", "k8s", "Id of the cluster. Shown on slack alerts.")
```

Comment:
Change it to `cluster-name`, and lgtm.

(It's a pity the "real" `metadata.clusterName` field never became useful)